charts/backstage/values.yaml

```diff
@@ -39,7 +46,40 @@ upstream:
         secretKeyRef:
           key: postgres-password
           name: "{{ .Release.Name }}-postgresql"
-
+    extraVolumeMounts:
+      - mountPath: /opt/app-root/src/dynamic-plugins-root
+        name: dynamic-plugins-root
+      - mountPath: /opt/app-root/src/dynamic-plugins-root/dynamic-plugins.yaml
+        name: dynamic-plugins
+        readOnly: true
+        subPath: dynamic-plugins.yaml
+    extraVolumes:
+      - emptyDir:
+          name: dynamic-plugins-root
+        name: dynamic-plugins-root
+      - configMap:
+          defaultMode: 420
+          name: dynamic-plugins
+          optional: true
+        name: dynamic-plugins
+    initContainers:
+      - command:
+          - python
+          - install-dynamic-plugins.py
+          - /dynamic-plugins-root
+        image: >-
+          image-registry.openshift-image-registry.svc:5000/developer-tests/backstage-showcase:latest
```

Comment:
As you probably know already, this will not work 🙂 (just stating the obvious, so we don't overlook it)

Reply:
Yeah, sure. That's part of why the PR is still WIP :-)
But that's good that you mentioned it, so that we don't forget changing it before merging !